spec/fixtures/ocr/bb222cc3333.xml.result.xml

```diff
@@ -55,7 +55,7 @@
       UseFindAndFilterTypographicRaster="false" UseSmartColorityDetection="false"
```

Comment:
These are important changes to get the actual file paths from Abbyy to test that they get mapped to local unix directories.